tools/ci_build/github/linux/docker/Dockerfile.package_ubuntu_2404_gpu

```diff
@@ -46,17 +49,17 @@ RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/
     libnvonnxparsers-dev=${TRT_VERSION} \
     libnvonnxparsers10=${TRT_VERSION} \
     tensorrt-dev=${TRT_VERSION} \
-    libnvinfer-bin=${TRT_VERSION} &&\
-    if [ $(echo $CUDA_VERSION | cut -d"." -f1) -ge 12 ]; then apt-get install -y cudnn9-cuda-12 ; fi
-#   ^^^^^^^^^^^If cuda version is 12 or higher, install cudnn 9 for cuda 12
+    libnvinfer-bin=${TRT_VERSION} && \
+    rm -rf /var/lib/apt/lists/*
 
-ADD scripts /tmp/scripts
+COPY scripts /tmp/scripts
 RUN cd /tmp/scripts && /tmp/scripts/install_dotnet.sh && rm -rf /tmp/scripts
 
 # Build final image from base.
-FROM base as final
+FROM base AS final
 ARG BUILD_USER=onnxruntimedev
 ARG BUILD_UID=1000
-RUN adduser --uid $BUILD_UID $BUILD_USER
+RUN adduser --uid $BUILD_UID $BUILD_USER --disabled-password --gecos "ONNX Runtime Developer" \
```

Comment:
Ubuntu 24.04's base image already has a builtin user(the previous versions did not have). So, if the builtin user's uid is 1000 and here if BUILD_UID is also 1000, this line of code will fail.